mrp_production_component_report_xlsx/__manifest__.py

```
@@ -0,0 +1,17 @@
+    "summary": "Print aggregated component list for selected "
+    "Manufacturing Orders to XLSX",
+    "version": "18.0.1.0.0",
+    "category": "Manufacturing",
+    "author": "Cetmix, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture-reporting",
+    "license": "AGPL-3",
+    "depends": ["mrp", "report_xlsx_helper"],
+    "data": [
+        "report/component_report_xlsx.xml",
+    ],
+}
```

Pylint (4.0.4)

[warning] Statement seems to have no effect

(W0104)